src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -889,6 +1076,24 @@ jsBackend.mediaLibraryHelper.upload =
     },
 
     /**
+    * Configure the uploader to trigger the cropper
```

Comment:
A missing space before the `*` if you want to make it consistent with the phpdocs